Makefile

```diff
@@ -67,3 +67,16 @@ uptest: $(UPTEST) $(KUBECTL) $(KUTTL)
 e2e: build controlplane.up local.xpkg.deploy.configuration.$(PROJECT_NAME) uptest
 
 .PHONY: uptest e2e
+
+local.xpkg.sync: local.xpkg.init $(UP)
+	@$(INFO) enable crossplane args
+	${KUBECTL} -n $(CROSSPLANE_NAMESPACE) patch deployment crossplane --type=json -p='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--enable-environment-configs"}]'
```

Comment:
it looks like not a very generic way to add custom args to crossplane :)

Reply:
i know - will open a PR in build submodule

Reply:
@ytsarev  please check: https://github.com/upbound/build/pull/237 if merged i will update PR

Reply:
done